### ISSUE TITLE
[Request] Solus distributor logo

### ISSUE THREAD

**Issue by feskyde** [keeferrourke/la-capitaine-icon-theme#66]
Requesting distributor icons for [Solus](https://solus-project.com)

Icon name: `distributor-logo-solus` and `start-here-solus`
Icon reference:
![distributor-logo-solus](https://cloud.githubusercontent.com/assets/20542037/21296152/6d7c8bda-c543-11e6-8fc8-4d444c866f9c.png)

**Comment by keeferrourke** [comment]
can you paste the output of `cat /etc/os-release` and `lsb_release -si` from a Solus box?

**Comment by feskyde** [comment]
```bash
casa@solaris ~> cat /etc/os-release
NAME="Solus"
VERSION=1.2.1
ID=solus
VERSION_ID=1.2.1
PRETTY_NAME="Solus 1.2.1"
ANSI_COLOR="1;34"
HOME_URL="https://www.solus-project.com"
SUPPORT_URL="https://www.solus-project.com/forums/"
BUG_REPORT_URL="https://dev.solus-project.com/"
casa@solaris ~> lsb_release -si
Solus
```

**Comment by keeferrourke** [comment]
Perfect! thanks, I'll push an update sometime soon, no guarantees on when though. Please bear with me :)

**Comment by JoshStrobl** [comment]
For that matter, Budgie support would be appreciated as well.

**Comment by keeferrourke** [comment]
@JoshStrobl what would be required for proper Budgie support? I don't currently have the time or resources the test the desktop environment, or do a Solus install at the moment. Feel free to open a new issue to address it if you want.

**Comment by JoshStrobl** [comment]
Filed #67 with more details, let me know if you need any more or testing, happy to help.

**Comment by keeferrourke** [comment]
Alright, so for proper Solus support:

* make `distributor-logo-solus.svg`
* add `application-x-solus-package` mime type for *.eopkg files
* add alternative mime type for *.desktop files
* address concerns in #67 
* update `configure`

am I missing anything? I'll be pushing updates soon&trade;

**Comment by keeferrourke** [comment]
@feskyde here's a preview of the distributor-logo!

![solus](https://cloud.githubusercontent.com/assets/6888381/21468695/d075ec18-c9ee-11e6-8c23-302a6b994ef0.png)
`distributor-logo-solus`

**Comment by JoshStrobl** [comment]
Looks great @keeferrourke. You're not missing anything that I can think of off to the top of my head :+1:

**Comment by keeferrourke** [comment]
with proper support for Budgie, as addressed in issue #67, I've pushed commit 54e980f which adds support for Budgie and Solus in the `configure` script. Solus should now be fully supported!

I'm going to close this issue for now, but if I missed something feel free to ping me and I'll reopen it.

**Comment by feskyde** [comment]
Thanks! 😄
